### ISSUE TITLE
Node 12 issue

### ISSUE THREAD

**Issue by icebob** [pvarentsov/iola#3]
Hi,

It doesn't work on Node 12.22

```
> iola io ws://localhost:3000

D:\Work\moleculer\moleculer-io\node_modules\iola\dist\core\common\util\message.util.js:85
            maxArrayLength: options?.maxArrayLength || 16,
                                    ^

SyntaxError: Unexpected token '.'
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (D:\Work\moleculer\moleculer-io\node_modules\iola\dist\core\common\index.js:20:14)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! moleculer-io@1.1.3 dev:client: `iola io ws://localhost:3000`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the moleculer-io@1.1.3 dev:client script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Icebob\AppData\Roaming\npm-cache\_logs\2022-01-09T16_08_28_615Z-debug.log

```

**Comment by pvarentsov** [comment]
Hi @icebob, 

Thanks for the issue. I will fix it tomorrow.

**Comment by pvarentsov** [comment]
Hi @icebob. Fixed it in v0.5.7.

**Comment by icebob** [comment]
Thanks
